trunner/runners/common.py

```diff
@@ -78,9 +80,11 @@ def unbind_rpi_usb(port_address):
 class Psu:
     """Wrapper for psu program"""
 
-    def __init__(self, script, cwd=_BOOT_DIR):
-        self.script = script
+    def __init__(self, target, script, cwd=None):
+        if cwd is None:
+            cwd = boot(target)
```

Comment:
so You're booting the target here? :P

please change the function name to at least `boot_dir`

Reply:
Yeah, `boot` isn't the best name :P. Changed to `boot_dir` as suggested.